src/transformers/generation/logits_process.py

```diff
@@ -19,6 +19,7 @@
 
 import numpy as np
 import torch
+import torch.nn.functional as F
```

Comment:
```suggestion
import torch.nn.functional as F
```

let's keep the same verbose import for consistency :)